### PR TITLE
GCP Whitelist Feed

### PR DESCRIPTION
**Pull request by cs-omkar-kumbhar**
Description:

PMDB Link: https://pmdb.fortinet.com/ProjectManagement/project/detail/18051

UTCs:
Connector installation verified.
Connector logo verified.
Actions and Playbook list verified.
Playbook tags verified.
All Playbook are in info mode verified.
All Playbook are in inactive mode verified.

Verified all actions.

### CODE REVIEW

File: google-cloud-platform-whitelist-feed/connector.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from connectors.core.connector import get_logger, ConnectorError, Connector
+from .operations import operations, check_health_ex
+
+logger = get_logger("google-cloud-platform-whitelist-feed")
+
+
+class GoogleCloudPlatformWhitelistFeed(Connector):
+    def execute(self, config, operation, params, **kwargs):
+        try:
+            connector_info = {"connector_name": self._info_json.get("name"),
```
Comment (cs-renuka-andhale) [inline google-cloud-platform-whitelist-feed/connector.py]: Is this code required ?

File: google-cloud-platform-whitelist-feed/connector.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                              "connector_version": self._info_json.get("version")}
+            operation = operations.get(operation)
+        except Exception as err:
+            logger.exception(err)
+            raise ConnectorError(err)
+        return operation(config, params)
+
+    def check_health(self, config):
+        logger.info("starting health check")
+        connector_info = {"connector_name": self._info_json.get("name"),
```
Comment (cs-renuka-andhale) [inline google-cloud-platform-whitelist-feed/connector.py]: Is this code required ?

File: google-cloud-platform-whitelist-feed/info.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "google-cloud-platform-whitelist-feed",
+  "version": "1.0.0",
+  "label": "Google Cloud Platform Whitelist Feed",
+  "category": "Network Security",
```
Comment (cs-prerna-joshi) [inline google-cloud-platform-whitelist-feed/info.json]: update the category

File: google-cloud-platform-whitelist-feed/info.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  "publisher": "Fortinet",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "help_online": "",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "ingestion_supported": true,
+  "ingestion_modes": [
+    "scheduled"
+  ],
```
Comment (cs-prerna-joshi) [inline google-cloud-platform-whitelist-feed/info.json:16]:
```suggestion
  ],
  "ingestion_preferences": {
    "modules": ["threat_intel_feeds", "indicators"]
  },
  "tags": [
    "ThreatIntel"
  ],
```

**Pull request merged**